pkg/networkpath/traceroute/tcp/utils.go

```diff
@@ -23,6 +24,14 @@ type (
 		DstIP       net.IP
 		TCPResponse layers.TCP
 	}
+
```

Comment:
Why is this in `utils`? I think this could well be in its own file, e.g. `tcp_parser.go`.

Reply:
Moved this out to a file called `parser.go` since we're already in the TCP package